### DISCUSSION TITLE
New Achievements

### DISCUSSION THREAD

**Discussion by EpiCreeper** [Quaver/Quaver#4765] [Feature Request]
While not gameplay related, this is technically a feature request, so I'm writing it down here.

Most of Quaver's achivements are related to gameplay, but I think there's a lot of room for achivements that dont relate directly to how good you are at vsrgs. Here is a list of ideas I had that could make for more different achivements [anything in square brackets is a name suggestion]:
- Downloading a skin from the workshop [Ohh Pretty!]
- Uploading a map to the server [Welcome To The Internet]
- Talking to QuaverBot in the in game chat [Quav GPT]
- Using QuaverBot commands in mp lobbies [Hackerman]
- Enabling the No SV mod [Epilepsy Warning]
- Dying during the last 10 seconds of a mapset [So Close!]
- Playing in a mp lobby with someone with a rank different then donator (rs, constributor, dev...) [Paparazzi]
- Playing in a mp lobby with at least one player of each status (host, ready, map downloaded, map not downloaded), [UNO]
- Play with a scroll speed of over 50 [Note Go BRRRR]
- Pass a map with split scroll enabled [Why Not Both?]
- Enable lane cover [Darker Yet Darker]
- Be AFK for an hour while Quaver is open [*Snore* mimimimimi]
- Make a playlist [24 Hour Chill Maps Radio For Button Pressing]
- Download a playlist [Discounted Bundle]
- Play a map in x2 speed [I Am Speed]
- Play a map with FLN / inverse modifire enabled [I Wish To BeLong]
- Play a map with NLN enabled [Short And Sweet]
- Pass a map with a single miss [L]

Alright thats all the ideas I have! Some of the names are kinda crap but I'm sure you could think of better names for some of those. I just wanna have more achivements that don't really mean anything but are just there to be goofy, although I think some of those teach new players about the game a little.

**Comment by Swan** [comment]
I don't mind adding new achievements, as long as they're not too quirky and could also be validated by our server.

> Uploading a map to the server
> Play a map with NLN enabled 

Something like this is good

> Be AFK for an hour while Quaver is open 
> Playing in a mp lobby with someone with a rank different then donator 

Something like this probably not lol

**Comment by ItzKoshepen** [comment]
To further add onto EpicCreeper's list of achievements suggestions, I'd like to suggest more achievements and sort them into different type of achievements:

### **Introductory**

These are achievements that congratulate new players for exploring and trying out features within Quaver, and does not require any form of skills at all to achieve

- Enter a multiplayer lobby for the first time [Hello World]
- Finish a map with unranked mods applied [Just For Fun]
- Chat in any channels for the first time [Socialist]
- Import/Drag any maps from VSRGs other than Quaver into Quaver [How did this work?]
- Adjust scroll speed to any value other than the default value [Speedometer]
- Select any judgement windows other than the Standard* window [Making a difference]
- Create a new custom judgement window [Customization]
- Friend a player [Party Animal]
- Mutual a player [Mutual Acknowledgement]
- Enter the download map tab ingame [There's more!?]
- Download an unranked map [Risky downloads]
- Use the view map function [Foreseer]
- Search map with any criteria searches (l>200, ln>50%, etc.) [Picky Eater]
- Set a custom FPS [Silky Smooth]

### **Skill related**

As the name suggests, these are achievements that are given to players who have reached a certain skill level and is able to achieve certain plays

- Get a 10:1 or higher ratio [Acc monster] (This achievement can be expanded to higher ratios]
- Passing a map with SV within the map [Notes can jump?]
- Pass a map with <1:1 ratio [Pushing much?]
- Pass a map with length>10 minutes [Eyes tired yet?]

### **Gimmicky**

These achievements are not usually achievable with normal gameplay, but rather achieved by playing the game specifically for the sole purpose of achieving the achievement (But of course may be achievable under very random or specific normal gameplay circumstances)

- Pass a map with 5 mods enabled (Highest number of mods possible without co-op because co-op is kinda buggy) [Full Flavor]
- Play Quaver for 5 (or literally any number) hours [Time for a break...?]
- Have more than 10,000 maps installed on Quaver (Unranked, unsubmitted and other games included) [Disk Space Destroyer]
- Be in a full (16 players) multiplayer lobby [Quite Crowded in Here]
- Get messaged by QuaverBot [AI Domination]
- Participate in a Quaver Official Tournament [Competitor]

**Comment by the-Shucks** [comment]
5,20,40,45,50 PR achievements PLEASE
